docs/getting_started/examples.rst

```diff
@@ -9,26 +9,26 @@ Usage Examples
 
 The following notebooks demonstrate how Modin can be used for scalable data science:
 
-- Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]
-- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/Modin_Taxi.ipynb>`_]
-- Modin for Machine Learning with scikit-learn [`Source <https://github.com/modin-project/modin/blob/master/examples/modin-scikit-learn-example.ipynb>`_]
+- Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`__]
```

Comment:
We should replace named links (single underscores) with anonymous links (double underscores) since presence of multiple named links with the same names in the single rst file causes sphinx warnings.

Reply:
What are the implications of using anonymous links?

Reply:
> What are the implications of using anonymous links?

Implication is that you can't reference to the target no more, for example with non-anonymous links the following is possible: you create a target `link to google` with an actual url specified and use it then just reference with no url
```rst
`link to google <https://google.com>`_

`link to google`_ # also links to google
```
however, adding another target called the same generates a warning:
```rst
`link to google <https://google.com>`_

`link to google <https://google.ru>`_ # a warning here as the names are duplicated
```
that's why the links were replaced to anonymous ones, with anon links you can create multiple targets with the same name, but can't reference then no more:
```rst
`link to google <https://google.com>`__
`link to google <https://google.ru>`__

`link to google`_ # invalid link
`link to google`__ # invalid link
```

Reply:
Answered [here](https://github.com/modin-project/modin/pull/4297#discussion_r841601488).